node-runner-cli/tests/fixtures/docker-config.yaml

```diff
@@ -19,7 +19,7 @@ core_node:
   enable_transaction: 'true'
   java_opts: --enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m
     -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts
-    -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
+    -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.log_util.log4j.core.async.AsyncLoggerContextSelector
```

Comment:
I think this is not write as the string replaced is a java option and nothing to do with nodecli logging

Reply:
ah! good catch! this was the IDE automatically renaming stuff. reverting